src/components/Charts/utils.ts

```diff
@@ -163,6 +163,7 @@ const getZoneLabels = (
     const toValue = zone.value || maxYAxis;
     const isActive = fromValue <= value && value < toValue;
     const activeClassName = isActive ? 'ZoneAnnotation--isActive' : '';
+    console.log(zone.labelClassName, activeClassName);
```

Comment:
Stale console.log?

Reply:
removed in my "removing the console log, my true enemy" commit